config/dependency.go

```diff
@@ -1,34 +1,34 @@
 package config
 
 import (
-	"bufio"
-	"bytes"
-	"encoding/json"
-	"fmt"
-	"path/filepath"
-	"strings"
-
-	"github.com/hashicorp/hcl2/hcl"
-	"github.com/zclconf/go-cty/cty"
-	"github.com/zclconf/go-cty/cty/gocty"
-	ctyjson "github.com/zclconf/go-cty/cty/json"
-
-	"github.com/gruntwork-io/terragrunt/errors"
-	"github.com/gruntwork-io/terragrunt/options"
-	"github.com/gruntwork-io/terragrunt/util"
+  "bufio"
+  "bytes"
+  "encoding/json"
+  "fmt"
+  "path/filepath"
+  "strings"
+
+  "github.com/hashicorp/hcl2/hcl"
+  "github.com/zclconf/go-cty/cty"
+  "github.com/zclconf/go-cty/cty/gocty"
+  ctyjson "github.com/zclconf/go-cty/cty/json"
+
+  "github.com/gruntwork-io/terragrunt/errors"
+  "github.com/gruntwork-io/terragrunt/options"
+  "github.com/gruntwork-io/terragrunt/util"
```

Comment:
Hmm these changes to the file tabs don't look correct?

Reply:
fixed.